data/data/powervs/iaas/power-iaas.tf

```diff
@@ -0,0 +1,22 @@
+provider "ibm" {
+ ibmcloud_api_key = var.ibmcloud_api_key
+}
+
+data "ibm_resource_group" "group" { 
+  name = var.cloud_resource_group
+}
+
+resource "ibm_resource_instance" "resource_instance" { 
+  name     = var.service_instance_name
+  service  = "power-iaas"
+  plan     = var.cloud_plan
+  location = var.ibmcloud_region
+  tags     = var.service_tags 
```

Comment:
Lets add the default tag by using the cluster_id like happens in the aws, e.g: https://github.com/openshift-powervs/installer/blob/add-ipi/data/data/aws/vpc/vpc.tf#L13:L17

Reply:
I added two tags, because of a comment below  "${var.cluster_id}-power-iaas", "${var.cluster_id}"